test.sh

```diff
@@ -24,3 +26,19 @@ yarn test
 
 # Run end-to-end tests
 # TODO
+
+# Upload code coverage
+if [[ "${JOB_TYPE}" == "presubmit" ]]; then
+       echo "detected PR code coverage job for #${PULL_NUMBER}"
+       REF_FLAGS="-P ${PULL_NUMBER} -C ${PULL_PULL_SHA}"
+elif [[ "${JOB_TYPE}" == "postsubmit" || "${JOB_TYPE}" == "periodic" ]]; then
```

Comment:
We could add an env variable in the postsubmit script and check for it here. But I wouldn't spend time on it, since if we were to switch CI/CD we'd have to change this script accordingly anyway.

Reply:
I agree with @florkbr to put the Prow CI workflow specific bits into a separate script, but that can be done as a follow-up.